install/resources/settings.xml

```diff
@@ -234,7 +234,8 @@ email=Sent via Email]]></optionscode>
 			<description><![CDATA[Here you can set the number of next and previous page links to show in the pagination for threads or forums with more than one page of results.]]></description>
 			<disporder>12</disporder>
 			<optionscode><![CDATA[numeric]]></optionscode>
-			<settingvalue><![CDATA[5]]></settingvalue>
+			<settingvalue><![CDATA[5
```

Comment:
Must've been pretty late when I've done it. Going to change it with the next commit